pkgs/development/arduino/arduino-mk/default.nix

```diff
@@ -11,8 +11,16 @@ stdenv.mkDerivation rec {
     sha256 = "0flpl97d2231gp51n3y4qvf3y1l8xzafi1sgpwc305vwc2h4dl2x";
   };
 
-  phases = ["installPhase"];
-  installPhase = "ln -s $src $out";
+  nativeBuildInputs = with python2Packages; [ wrapPython ];
+  propagatedBuildInputs = with python2Packages; [ pyserial ];
+  installPhase = ''
+    mkdir -p $out/share/man/man1
+    cp -rT $src $out
```

Comment:
I think using installManPage here would be easer.

Reply:
Thanks